tensorboard/webapp/feature_flag/store/feature_flag_reducers_test.ts

```diff
@@ -108,6 +108,149 @@ describe('feature_flag_reducers', () => {
     });
   });
 
+  describe('#featureFlagOverrideChanged', () => {
+    it('does not remove existing overrides', () => {
+      const prevState = buildFeatureFlagState({
+        flagOverrides: {
+          inColab: true,
+          defaultEnableDarkMode: true,
+        },
+      });
+      const nextState = reducers(
+        prevState,
+        actions.featureFlagOverrideChanged({
+          flags: {enabledLinkedTime: true},
+        })
+      );
+      expect(nextState.flagOverrides).toEqual({
+        inColab: true,
+        defaultEnableDarkMode: true,
+        enabledLinkedTime: true,
+      });
+    });
+
+    it('changes values of existing overrides', () => {
+      const prevState = buildFeatureFlagState({
+        flagOverrides: {
+          inColab: true,
+          defaultEnableDarkMode: true,
+        },
+      });
+      const nextState = reducers(
+        prevState,
+        actions.featureFlagOverrideChanged({
+          flags: {inColab: false},
+        })
+      );
+      expect(nextState.flagOverrides).toEqual({
+        inColab: false,
+        defaultEnableDarkMode: true,
+      });
+    });
+  });
+
+  describe('#featureFlagOverridesReset', () => {
+    it('does nothing when no overrides are provided', () => {
+      const prevState = buildFeatureFlagState();
+      const nextState = reducers(
+        prevState,
+        actions.featureFlagOverridesReset({flags: []})
+      );
+      // Intentionally using toBe rather than toEqual to ensure the object is the SAME object (not a copy).
+      expect(nextState).toBe(prevState);
+    });
+
+    it('does not modify previous state', () => {
```

Comment:
FWIW we shouldn't be modifying previous state in any of our reducers but we never test for it. You could consider dropping this test.

Reply:
I'm aware that the previous state should never be modified but I figured it was worth testing. I guess if that's opposed to convention though I'll remove the test.